### PR TITLE
Added min function

### PR DESCRIPTION
**Pull request by lyfofvipin**
Python implementation of the Python Standard Library must have all the inbuilt  available  within it
It's min function if it got successfully added then i will try to code some more. :)

### CODE REVIEW

File: ouroboros/__builtins__.py [hunk 1]
```diff
@@ -135,6 +135,30 @@ def __exit__(self, type, value, tb):
...
 
+#min
+def min(object):
+	"""
+		This method will take an iterator and return the minimum value from there.
+	"""
+	if type(object) is int:
+		raise TypeError("'int' object is not iterable")
+	elif type(object) is float:
+		raise TypeError("'int' object is not iterable")
```
Comment (gleydson404) [inline ouroboros/__builtins__.py:146]: This line, should be:

`raise TypeError("'float' object is not iterable")`

**Review by gleydson404 — CHANGES_REQUESTED** [review]

**Comment by phildini** [comment]
Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue, or think about directing your efforts to https://github.com/beeware/briefcase or https://github.com/beeware/toga. Both of these have more active development right now. 😄

File: ouroboros/__builtins__.py (hunk 1 above)
Comment (lyfofvipin) [reply ouroboros/__builtins__.py:146]: Yes my bad will fix it soon.

**Comment by lyfofvipin** [comment]
Acknowledged

**Comment by PythonLinks** [comment]
I hope that development on this library continues.  I will be needing it in a few months, maybe sooner, more likely later.

**Pull request closed without merging**